packages/plugin-core/src/utils/StartupPrompts.ts

```diff
@@ -36,17 +36,18 @@ export class StartupPrompts {
       );
     }
 
+    const timeFromFirstInstall = CUR_TIME.minus(
```

Comment:
What's the functional difference here? Looks the same to me, but maybe my eyes aren't working.

Reply:
same, just turned into variables to make it a little more readable

Reply:
Ah ok - so the 'fix lapsed user message being shown to new users' part of the change comes from changing the 'break' to a return' statement - got it.